src/SettingView.ts

```diff
@@ -359,29 +376,32 @@ export default class SettingView {
 
 		themeModal.contentEl
 			.createEl("h2", { text: "Theme Settings" })
-			.prepend(getIcon("palette"));
+			.prepend(this.getIcon("palette"));
+
+		const themesListResponse =
+			await axios.get<IObsidianTheme[]>(OBSIDIAN_THEME_URL);
 
-		const themesListResponse = await axios.get(
-			"https://raw.githubusercontent.com/obsidianmd/obsidian-releases/master/community-css-themes.json",
-		);
 		new Setting(themeModal.contentEl).setName("Theme").addDropdown((dd) => {
 			dd.addOption('{"name": "default", "modes": ["dark"]}', "Default");
+
 			const sortedThemes = themesListResponse.data.sort(
-				(a: { name: string }, b: { name: any }) =>
+				(a: { name: string }, b: { name: string }) =>
 					a.name.localeCompare(b.name),
 			);
-			sortedThemes.map((x: any) => {
+
+			sortedThemes.map((x) => {
 				dd.addOption(
 					JSON.stringify({
 						...x,
 						cssUrl: `https://raw.githubusercontent.com/${x.repo}/${
+							// the link to themes seems to never have "branch"
 							x.branch || "HEAD"
```

Comment:
didn't see a single branch under the theme-output, does it ever exist?

Reply:
According to obsidian documentation it can exists. But I suspect almost no one is using it, and just have their published theme in the default branch.